### PR TITLE
Refactor - Rich Formatting dropdown

### PR DESCRIPTION
**Pull request by e1himself**
- Disconnect Rich Formatting dropdown options rendering from the specific Elements components (less styles to override)
- Dramatically simplify Dropdown options rendering logic
- Change how Dropdown symbols are exported (it's a TS namespace now)
- Convert all related stylesheets to SCSS modules
- No changes in visual presentations are expected

### CODE REVIEW

File: packages/slate-editor/src/components/Menu/Dropdown.module.scss [hunk 1]
```diff
@@ -132,54 +132,4 @@
...
-            padding: 0;
-            background: unset;
-        }
-
-        &--blockquote > p {
-            padding-left: $spacing-half;
-            font-style: italic;
-        }
-
-        &--blockquote > p::before {
```
Comment (e1himself) [inline packages/slate-editor/src/components/Menu/Dropdown.module.scss:177]: This part was a spaghetti blob mixed with the code located on a different abstraction level. Fixed by moving the styles around to where they belong.

File: .eslintrc.cjs [hunk 2]
```diff
@@ -62,6 +62,7 @@ module.exports = {
         "@typescript-eslint/consistent-type-imports": "error",
         "@typescript-eslint/no-empty-interface": "off",
         "@typescript-eslint/no-explicit-any": "off",
+        "@typescript-eslint/no-namespace": "off",
```
Comment (aspirisen) [inline .eslintrc.cjs:65]: 👍

File: packages/slate-editor/src/components/Menu/Dropdown.tsx [hunk 3]
```diff
@@ -1,31 +1,35 @@
...
+        hidden?: boolean;
+        label: string;
+        value: Value;
+    }
 
-export interface Props<Value extends string> extends Omit<DropdownProps, 'onChange'> {
-    onChange: (value: Value) => void;
-    options: Option<Value>[];
-    value?: Value;
+    export interface Props<Value extends string> extends Omit<DropdownProps, 'onChange'> {
```
Comment (aspirisen) [inline packages/slate-editor/src/components/Menu/Dropdown.tsx:16]: Is it new approach that we should follow from now?

**Review by aspirisen — APPROVED** [review]

**Comment by aspirisen** [comment]
asd

File: packages/slate-editor/src/components/Menu/Dropdown.tsx (hunk 3 above)
Comment (e1himself) [reply packages/slate-editor/src/components/Menu/Dropdown.tsx:16]: Yes. No. I don't know. Let's talk :) 

I wanted to group the dropdown with its related types, and tried this approach. It's nicer than prefixing all the types with `Dropdown`. Nicer than ES namespaces (though not native). 

What do you think? Yay or neigh?

Comment (aspirisen) [reply packages/slate-editor/src/components/Menu/Dropdown.tsx:16]: I would like to give it a try, for now i do not see disadvantages 😊

File: packages/slate-editor/src/components/Menu/Dropdown.module.scss [hunk 4]
```diff
@@ -52,9 +52,23 @@
...
+                content: "";
+                display: block;
+                position: absolute;
+                left: $spacing-base;
+                top: $spacing-base;
+                bottom: $spacing-base;
+                right: $spacing-base;
+                border-radius: $border-radius-base;
+                background-color: $toolbar-dark-theme-hover-bg;
+            }
```
Comment (e1himself) [inline packages/slate-editor/src/components/Menu/Dropdown.module.scss:68]: It turned out that the hover state was missing here, and instead was applied from the main app stylesheet :grimacing:

**Pull request merged**